--- iotop.py.orign	2007-11-03 18:46:19.000000000 +0100
+++ iotop.py	2007-12-28 14:37:05.000000000 +0100
@@ -440,9 +440,12 @@
         if not self.options.batch:
             self.win = win
             self.resize()
-            curses.use_default_colors()
-            curses.start_color()
-            curses.curs_set(0)
+	    try:
+	            curses.use_default_colors()
+        	    curses.start_color()
+        	    curses.curs_set(0)
+	    except:
+        	    pass		
 
     def resize(self):
         self.height, self.width = self.win.getmaxyx()
